Library/Homebrew/dev-cmd/vendor-gems.rb

```diff
@@ -40,7 +40,7 @@ def vendor_gems
       end
 
       ohai "bundle install --standalone"
-      safe_system "bundle", "install", "--standalone"
+      safe_system_redirect_stdout_to_stderr "bundle", "install", "--standalone"
```

Comment:
I'm not sure about using these methods just to be used in a single place each (unless you have other ideals). I'd be tempted to just inline the methods?

Reply:
Yeah agreed that two single-use methods are overkill. I went with combining into one method, that is now used in both places. I suspect there are other invocations of bundler or other tools that would need this, so having a function for it seems probably worth it, what do you think?